src/components/Orders/Orders.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import useCurrency from "../hooks/useCurrency";
+import MonthlySales from "../totoalSales/monthlySales";
+import LifetimeSales from "../totoalSales/lifetimeSales";
+import * as actionCreators from "../../state/actionCreators";
+import { Table, Tag, Button } from "antd";
+import { connect } from "react-redux";
+import "antd/dist/antd.css";
+
+// import { data } from "../../db.js";
```

Comment:
Please remove the code that is commented out. If its not needed then you can delete it and readd if needed.